handlers/user.go

```diff
@@ -1723,16 +1723,20 @@ func MultipleUsersNotificationsSubscribe(w http.ResponseWriter, r *http.Request)
 		EventThreshold float64 `json:"event_threshold"`
 	}
 
+	errFields := map[string]interface{}{
+		"route": r.URL.String(),
+	}
```

Comment:
You'll probably want to include the request body here too, since this is a POST request there will be relevant data in there too. Same goes for `func MultipleUsersNotificationsUnsubscribe`